.changeset/poor-windows-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fake-scope/fake-pkg": patch
```

Comment:
what happened here 😕

Reply:
Oh good catch! 🤔

Reply:
I just used the generated default value and left it for you to catch 😄